wikimap/data.py

```diff
@@ -0,0 +1,80 @@
+import itertools
+import json
+from xlrd import open_workbook
+import networkx as nx
+
+
+# READING
+
+# add caching decorator
```

Comment:
Why do you need a caching decorator? Caching for such a small and focused library seems like over-engineering, and it might introduce additional problems such as memory leaks.